source/User_Guide/Settings/reading_your_invoice.md

```diff
@@ -37,21 +34,19 @@ Need to send invoices to multiple addresses? Consider creating a [role address](
 
 These are the total charges for services used **in the current and previous month**: 
 
-![]({{root_url}}/images/invoice_NL_usage.png)
+![Charge summary section]({{root_url}}/img/invoice_NL_usage.png)
```

Comment:
For some reason this image didn't show up. Is this the one you accidentally deleted?